CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## v2.0.0
+Thu 23 Jan 17:07:52 GMT 2020
+
+* [6a9b8e3](https://github.com/hyperledger/fabric-chaincode-java/commit/6a9b8e3) [FABCJ-273](https://jira.hyperledger.org/browse/FABCJ-273) Release 2.0.0
```

Comment:
Probably shouldn't include the release 2.0 commit (it's not in yet and may end up with a different hash), and you're missing the squashed commit you just rebased on (which will highlight why squashing unrelated commits is evil).

Reply:
Yes;  very valid point.